### PR TITLE
Add codeowners for dependency track

### PR DESCRIPTION
**Pull request by generalova-kate**
Internal ticket 54581

### CODE REVIEW

File: .github/labeler.yml [hunk 1]
```diff
@@ -0,0 +1,5 @@
+# See help here: https://github.com/marketplace/actions/labeler
+
+dependency_changes:
```
Comment (eizamaliev) [inline .github/labeler.yml:3]: Please, use existing label `dependency`

**Review by eizamaliev — CHANGES_REQUESTED** [review]

File: .github/labeler.yml (hunk 1 above)
Comment (generalova-kate) [inline .github/labeler.yml:3]:
```suggestion
dependencies:
```

**Review by eizamaliev — APPROVED** [review]

**Comment by eaidova** [comment]
Jenkins please retry a build

**Comment by vladimir-dudnik** [comment]
10:36:41  running yamllint...
10:36:47  ./.github/labeler.yml
10:36:47    4:16      error    syntax error: expected alphabetic or numeric character, but found '*' (syntax)
10:36:47  
10:36:47  ./.github/workflows/labeler.yml
10:36:47    3:1       error    wrong indentation: expected 2 but found 0  (indentation)
10:36:47    9:5       error    wrong indentation: expected 6 but found 4  (indentation)
10:36:47    12:26     error    no new line character at the end of file  (new-line-at-end-of-file)
10:36:47  
10:36:47  running flake8...
10:36:57  running documentation checks...
10:36:58  .github/workflows/labeler.yml:12: last line doesn't end with a newline character
10:36:58  CODEOWNERS:5: last line doesn't end with a newline character

**Comment by generalova-kate** [comment]
Jenkins please retry a build

**Comment by generalova-kate** [comment]
Jenkins please retry a build

**Pull request merged**